com.bluejeans.BlueJeans.appdata.xml

```diff
@@ -35,6 +35,7 @@
     <content_attribute id="social-audio">intense</content_attribute>
   </content_rating>
   <releases>
+    <release version="2.21.2.1" date="2021-02-05"/>
     <release date="2018-10-07" version="1.37.22"/>
```

Comment:
Feel free to remove the old entries for versions 1.x.

Reply:
I might do that in the future, but I don't really see a need for it.